src/_common.scss

```diff
@@ -31,13 +31,14 @@ $tablet: '(min-width: 768px)';
     src: local('Lato Bold'), local('Lato-Bold'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Bold.woff2') format('woff2'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Bold.woff') format('woff');
 }
 
+@import '~box-react-ui/lib/styles/common/overlay';
```

Comment:
do we need to prefix this as well?

Reply:
The BRUI flyout unfortunately gets added directly to the body :-/